behaviours/meme_rater.py

```diff
@@ -153,11 +152,6 @@ async def rate_meme(
 
         if avg_rating >= MINIMUM_MEME_RATING_TO_NOT_DELETE:
             await message.add_reaction(emoji="👍")
-        elif DELETE_SHIT:
```

Comment:
We should keep this in. It's a distinct feature that would be useful if we wanted to turn it on

Reply:
I don't like dead code, if it's ever wanted it can be added in

Reply:
Can you comment it out? 

We can discuss KSCM Q2

Reply:
I could think of nothing worse